src/pages/patientView/PatientViewPage.tsx

```diff
@@ -135,28 +161,28 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         this.onCnaTableColumnVisibilityToggled = this.onCnaTableColumnVisibilityToggled.bind(this);
     }
 
+    @computed
+    private get studyIdAndCaseId():string {
```

Comment:
i would just put this stuff up in the reaction.

Reply:
Hmm. I was having trouble with the reaction firing last week, but now it seems to work fine with a simpler implementation, so yes, I will do this.